src/Backend/Modules/Faq/Actions/EditCategory.php

```diff
@@ -92,8 +92,8 @@ private function validateForm(): void
                 $item['id'] = $this->id;
                 $item['language'] = $this->record['language'];
                 $item['title'] = $this->form->getField('title')->getValue();
-                $item['extra_id'] = $this->record['extra_id'];
-                $item['meta_id'] = $this->meta->save(true);
+                $item['sequence'] = $this->record['sequence'];
```

Comment:
Don't change anything in the actions, only in the model.

Reply:
This is needed to make the update work, otherwise the sequence will be lost.
extra_id and meta_id are not needed in update

Reply:
Retrieve the item in the Model.php, for the current scope we decided to try to keep the current implementation in the model, so the methods in Model.php should be backwards compatible.

Reply:
Fixed 232348709